File: week1/day1/deaf_grandma/deaf_grandma.js
var prompt = require('prompt') ;
function deafGrandma( counter = 0 ) {
var output = 'Grandma is listening...'
prompt.start()
prompt.get( 'askGrandma', function(err,result) {
    if (counter < 3){
        var input = result.askGrandma
        if (input === ''){
            output='WHAT?!'
        }
        else if (input === /[a-z]/.test(input)){
            output='SPEAK UP KID'
        }
        else if (input === "GOODBYE"){
            counter ++
            if (counter == 1){ 
            output= "LEAVING SO SOON?"}
            if (counter == 2){
                output = 'LATER, SKATER'
            }
        }

        else if (input === input.toUpperCase()){
            output="NO, NOT SINCE 1946!"
        }
}
console.log(output)
if (counter < 2 ){
    deafGrandma(counter)
}
})
}

deafGrandma();

File: week1/day1/deaf_grandma/deaf_grandma.py
def deaf_grandma():
    print("HEY KID!")
    count=0
    yourInput=input("say something to grandma\n")
    while count !=1:
        if yourInput == 'GOODBYE!':
            print('LEAVING SO SOON?')
            yourInput=input("say something to grandma\n")
        if yourInput == 'GOODBYE!':
            print('LATER, SKATER!')
            count=1
            break
        if yourInput=="":
            print("WHAT?")
            yourInput=input("say something to grandma\n")
        if  yourInput != yourInput.upper():
            print("SPEAK UP KID")
            yourInput=input("say something to grandma\n")
        else:
            print("NO, NOT SINCE 1946")
            yourInput=input("say something else to grandma\n")
        

deaf_grandma()
